meta-sokol-flex-staging/recipes-kernel/lttng/lttng-tools/0002-Don-t-resolve-empty-strings-in-resolve_path.patch

```diff
@@ -0,0 +1,26 @@
+From 60bad37b01dfa851bc4b4a7fbb9a7086c7453251 Mon Sep 17 00:00:00 2001
+From: Christopher Larson <chris_larson@mentor.com>
+Date: Thu, 19 Oct 2023 11:08:18 -0500
+Subject: [PATCH] Don't resolve empty strings in resolve_path
+
+If a resolved path is set to the empty string, this would resolve it to
+$HOME. Instead, leave it as is so the caller can deal with it.
```

Comment:
it resolves to CWD, not $HOME.. we observed it to be $HOME because CWD was $HOME..

Also I am not really sure if we need to fix it.. which known problematic scenario are we fixing? if its really an issue, then IMHO it should go upstream.. just random thoughts so you can simply ignore it..

Reply:
Fair point, I'll look into submitting a few things in addition to this upstream, and drop the second patch in this PR, as I do think they should really have a cleaner error message if given incorrect consumerd paths.

Reply:
Fair point, I'll look into submitting a few things in addition to this upstream, and drop the second patch in this PR, as I do think they should really have a cleaner error message if given incorrect consumerd paths.